frontend/src/pages/Vulnerability/Vulnerability.tsx

```diff
@@ -461,6 +461,46 @@ export const Vulnerability: React.FC = () => {
                   </Timeline>
                 </div>
               </Paper>
+              <Paper className={classes.cardRoot}>
+                <div className={classes.inner}>
+                  <div className={classes.section}>
+                    <h2 className={classes.subtitle}>Provenance</h2>
+                    <p>
+                      <strong>Root Domain: </strong>
+                      {vulnerability.domain.fromRootDomain}
+                    </p>
+                    <p>
+                      <strong>Subdomain: </strong>
+                      {vulnerability.domain.name} (
+                      {vulnerability.domain.subdomainSource})
+                    </p>
+                    {vulnerability.service && (
+                      <p>
+                        <strong>Service/Port: </strong>
+                        {vulnerability.service.service
+                          ? vulnerability.service.service
+                          : vulnerability.service.port}{' '}
+                        ({vulnerability.service.serviceSource})
+                      </p>
+                    )}
+                    {['dnstwist', 'lookingGlass', 'hibp', 'shodan'].indexOf(
```

Comment:
Instead of hardcoding scan names, could we just check to see if `vulnerability.cpe` is defined?